charts/metal-control-plane/templates/metal-apiserver.yaml

```diff
@@ -0,0 +1,147 @@
+{{ if .Values.metal_apiserver.enabled }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: metal-apiserver
+  name: metal-apiserver
+spec:
+  replicas: {{ .Values.metal_apiserver.replicas }}
+  selector:
+    matchLabels:
+      app: metal-apiserver
+  template:
+    metadata:
+      labels:
+        app: metal-apiserver
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/metal-apiserver-config.yaml") . | sha256sum }}
+        checksum/masterdata-api-config: {{ include (print $.Template.BasePath "/masterdata-api-config.yaml") . | sha256sum }}
+    spec:
+      restartPolicy: Always
+      # it's better to disable the service links as the default args do not work properly otherwise (metal#64)
+      enableServiceLinks: False
+      containers:
+      - name: apiserver
+        image: {{ .Values.images.metal_apiserver.image }}:{{ .Values.images.metal_apiserver.tag }}
+        imagePullPolicy: {{ .Values.images.metal_apiserver.imagePullPolicy }}
+        ports:
+        - containerPort: 8080
+        - containerPort: 2112
+        # TODO: implement
+        # readinessProbe:
+        #   httpGet:
+        #     path: {{ .Values.metal_apiserver.base_path }}v1/health
+        #     port: 8080
+        #   initialDelaySeconds: 15
+        #   successThreshold: 1
+        #   timeoutSeconds: 10
```

Comment:
Since K8S v1.27 grpc liveness probe is a built in feature: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-grpc-liveness-probe

Reply:
Done.